tasks/seedManagement.js

```diff
@@ -8,9 +9,8 @@ task("sendTransactionSeed", "send transaction to Gnosis Safe")
       `Sending SeedFactory.deploySeed() transaction to ${safeAddress}`
     );
     const gnosis = api(safeAddress, network.name);
-    const { root } = await ethers.getNamedSigners();
```

Comment:
Can you explain why we don't need `root` anymore?

Reply:
`root` is the name of the first account that can be generated from the private key. Because we don't need a Signer (created from the PK) within this flow, we can remove the signer named `root`. Signing is done by the Singer contract